internal/matchers/temporal_matcher.go

```diff
@@ -16,6 +16,7 @@ package matchers
 
 type TemporalMatcher byte
 
+//nolint:revive
```

Comment:
I would like to know what was ignored and why
```suggestion
//nolint:revive // ignoring missing comments for unexported constants in an internal package
```